src/structures/Sticker.js

```diff
@@ -72,16 +105,62 @@ class Sticker extends Base {
     return new Date(this.createdTimestamp);
   }
 
+  /**
+   * Whether this sticker is partial
+   * @type {boolean}
+   * @readonly
+   */
+  get partial() {
+    return 'username' in this;
```

Comment:
Wait hold up, this doesn't exist on a sticker

Reply:
oops

Reply:
fixed